plugins/modules/scaleway_container.py

```diff
@@ -94,10 +94,18 @@
     type: dict
     default: {}
 
+  cpu_limit:
+    description:
+      - Resources define performance characteristics of your container.
+      - They are allocated to your container at runtime.
+      - Unit is 1/1000 of a VCPU
+    type: int
```

Comment:
```suggestion
    type: int
    version_added: 11.2.0
```
(Might be 11.3.0 since 11.2.0 will be released later today.)